docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a _Quick Fix_ automatic code replacement or by editing the code. You can use the *Delete* and *Mark as Complete* options to keep track of which issues you have finished working with. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.
+
+.Procedure
```

Comment:
I do not think this section works like a procedure. It's a list of options, not a group of steps that must be performed in a particular order. This would be a reference module. You could use a table or unordered list or even separate sections. The procedure format does not seem appropriate.